### DISCUSSION TITLE
Problem with select component

### DISCUSSION THREAD

**Discussion by amirsafaricg** [quasarframework/quasar#11876] [General - Components / Directives / etc] [answered]
whenever im using the https://quasar.dev/vue-components/select#customizing-menu-options select input 

when i click it, it looks like https://quasar.dev/vue-components/select#options-list-display-mode
the dialog coming from top

**Comment by MilosPaunovic** [comment, marked as answer]
Set `behaviour` prop of `QSelect` to either `dialog` or `menu`, based on your needs.

**Reply by amirsafaricg** [reply]
thank you :)))))
